options.go

```diff
@@ -58,13 +58,13 @@ func StreamBufferCapacity(capacity uint) func(Party) error {
 }
 
 // MaximumReceiveMessageSize is the maximum size of a single incoming hub message.
```

Comment:
please change to `// MaximumReceiveMessageSize is the maximum size in bytes of a single incoming hub message.`